verilog/tools/ls/autoexpand.cc

```diff
@@ -775,6 +790,18 @@ void AutoExpander::Module::PutDeclaredPort(const SyntaxTreeNode &port_node) {
                     std::move(unpacked_dimensions)});
 }
 
+bool AutoExpander::Module::DependsOn(
```

Comment:
This does not seem to always adhere to strict weak ordering.
In the [`Autoexpand.AUTO_ExpandPortsMultipleConnectionsWithAUTO_TEMPLATE` ](https://github.com/chipsalliance/verible/blob/398a8505af01f61e930bd95fc2a4f1131dc0d467/verilog/tools/ls/autoexpand_test.cc#L1850)
test, a [strict weak ordering check](https://discourse.llvm.org/t/rfc-strict-weak-ordering-checks-in-the-debug-libc/70217) shows that DependsOn() does not always result in a strict weak ordering in the sort using it:

https://github.com/chipsalliance/verible/blob/398a8505af01f61e930bd95fc2a4f1131dc0d467/verilog/tools/ls/autoexpand.cc#L1448-L1451